src/controller/cve.controller/index.js

```diff
@@ -86,10 +96,20 @@ router.get('/cve',
     description: 'A filtered list of CVE Records and pagination fields',
     content: {
       "application/json": {
-        schema: { $ref: '/schemas/cve/list-cve-records-response.json' }
+        schema: {
```

Comment:
This is a little confusing since the Published schema & example are for the whole record list, but the Rejected is just for a CVE record. I think we should just pull this out for now unless you can think of how to show the record list having some published and some rejected records.